docs/anomaly-detection/components/scheduler.md

```diff
@@ -145,11 +145,41 @@ Examples: `"50s"`, `"4m"`, `"3h"`, `"2d"`, `"1w"`.
             <td>str, Optional</td>
             <td>
 
-`"1h"`
+`1h`
             </td>
             <td>
 
-How often to completely retrain the models. If missing value of `infer_every` is used and retrain on every inference run.
+How often to completely retrain the models. If not set, value of `infer_every` is used and retrain happens on every inference run.
+            </td>
+        </tr>
+        <tr>
+            <td>
+
+`start_from`
+            </td>
+            <td>str, Optional</td>
+            <td>
+
+`2024-11-26T01:00:00Z`, `01:00`
+            </td>
+            <td>
+
+Available since [v1.18.5](https://docs.victoriametrics.com/anomaly-detection/changelog/#v1185). Specifies when to initiate the first `fit_every` call. Accepts either an ISO 8601 datetime or a time in HH:MM format. If the specified time is in the past, the next suitable time is calculated based on the `fit_every` interval. For the HH:MM format, if the time is in the past, it will be scheduled for the same time on the following day, respecting the `tz` argument if provided.
```

Comment:
```suggestion
Available since [v1.18.5](https://docs.victoriametrics.com/anomaly-detection/changelog/#v1185). Specifies when to initiate the first `fit_every` call. Accepts either an ISO 8601 datetime or a time in HH:MM format. If the specified time is in the past, the next suitable time is calculated based on the `fit_every` interval. For the HH:MM format, if the time is in the past, it will be scheduled for the same time on the following day, respecting the `tz` argument if provided. By default, the timezone defaults to UTC
```